feature/mpls/otg_tests/static_bgp_nexthop/static_bgp_nexthop_test.go

```diff
@@ -0,0 +1,447 @@
+	port2Ipv6.SetAddress(atePort2.IPv6).SetGateway(dutPort2.IPv6).SetPrefix(uint32(atePort2.IPv6Len))
+
+	port3Dev := topo.Devices().Add().SetName(atePort3.Name + ".dev")
+	port3Eth := port3Dev.Ethernets().Add().SetName(atePort3.Name + ".Eth").SetMac(atePort3.MAC)
+	port3Eth.Connection().SetPortName(port3.Name())
+	port3Ipv4 := port3Eth.Ipv4Addresses().Add().SetName(atePort3.Name + ".IPv4")
+	port3Ipv4.SetAddress(atePort3.IPv4).SetGateway(dutPort3.IPv4).SetPrefix(uint32(atePort3.IPv4Len))
+	port3Ipv6 := port3Eth.Ipv6Addresses().Add().SetName(atePort3.Name + ".IPv6")
+	port3Ipv6.SetAddress(atePort3.IPv6).SetGateway(dutPort3.IPv6).SetPrefix(uint32(atePort3.IPv6Len))
+	// Configure BGP on ATE Port 2 to advertise next-hop prefixes
+	bgproute := port2Dev.Bgp().SetRouterId(atePort2.IPv4)
+	bgpPeerv4 := bgproute.Ipv4Interfaces().Add().SetIpv4Name(port2Ipv4.Name()).Peers().Add().SetName("BGP4.peer2")
+	bgpPeerv4.SetPeerAddress(port2Ipv4.Gateway()).SetAsNumber(ateAS).SetAsType(gosnappi.BgpV4PeerAsType.EBGP)
+	bgpPeerv6 := bgproute.Ipv6Interfaces().Add().SetIpv6Name(port2Ipv6.Name()).Peers().Add().SetName("BGP6.peer2")
+	bgpPeerv6.SetPeerAddress(port2Ipv6.Gateway()).SetAsNumber(ateAS).SetAsType(gosnappi.BgpV6PeerAsType.EBGP)
+	bgpPeerv4Routes := bgpPeerv4.V4Routes().Add().SetName("BGP4.peer2.Route")
+	bgpPeerv4Routes.Addresses().Add().SetAddress(bgpNHv4).SetPrefix(24)
+
+	bgpPeerv6Routes := bgpPeerv6.V6Routes().Add().SetName("BGP6.peer2.Route")
+	bgpPeerv6Routes.Addresses().Add().SetAddress(bgpNHv6).SetPrefix(64)
+	iPv4MPLSForwarding(t, topo, ipv4Flow)
+	iPv6MPLSForwarding(t, topo, ipv6Flow)
+
+	ate.OTG().PushConfig(t, topo)
+	ate.OTG().StartProtocols(t)
+	otgutils.WaitForARP(t, ate.OTG(), topo, "IPv4")
+	otgutils.WaitForARP(t, ate.OTG(), topo, "IPv6")
+	return topo
+}
+
+func withdrawBGPRoutes(t *testing.T, routeNames []string) {
+	defer func() {
+		if r := recover(); r != nil {
+			t.Errorf("Failed to withdraw BGP routes: %v", r)
+		}
+	}()
+
+	ate := ondatra.ATE(t, "ate")
+	otg := ate.OTG()
+	cs := gosnappi.NewControlState()
+	cs.Protocol().Route().SetNames(routeNames).SetState(gosnappi.StateProtocolRouteState.WITHDRAW)
+	otg.SetControlState(t, cs)
+	t.Log("BGP routes withdrawn successfully")
+}
+
+// Add IPv4 MPLS Forwarding Flow
+func iPv4MPLSForwarding(t *testing.T, config gosnappi.Config, flowname string) {
+	dut := ondatra.DUT(t, "dut")
+	macAddress := gnmi.Get(t, dut, gnmi.OC().Interface(dut.Port(t, "port1").Name()).Ethernet().MacAddress().State())
+	flow := config.Flows().Add()
+	flow.SetName(flowname)
+	flow.Metrics().SetEnable(true)
+	flow.TxRx().Port().SetTxName(config.Ports().Items()[0].Name()).SetRxNames([]string{config.Ports().Items()[1].Name()})
+	eth := flow.Packet().Add().Ethernet()
+	eth.Src().SetValue(atePort1.MAC)
+	eth.Dst().SetValue(macAddress)
+
+	mpls := flow.Packet().Add().Mpls()
+	mpls.Label().SetValue(mplsLabelV4)
+
+	ip4 := flow.Packet().Add().Ipv4()
+	ip4.Src().SetValue(atePort1.IPv4)
+	ip4.Dst().SetValue(bgpNHv4)
```

Comment:
Shouldnt the flow destination be `ipv4Dst         = "203.0.113.0/24"` rather? This is a fundamental deviation from the Readme which requires "the traffic flow's destination to be a separate prefix that resolves via that BGP next-hop" and not the BGP next-hop itself.

Reply:
updated flow destination. Thanks!